lib/src/translate.service.ts

```diff
@@ -560,9 +564,9 @@ export class TranslateService {
      *
      * @returns string
      */
-    public getBrowserCultureLang(): string {
+    public getBrowserCultureLang(): string | null {
```

Comment:
You should make sure that in case `browserCultureLang` is `undefined`, `null` is returned.
(I.e. do something like `return browserCultureLang || null;` at the end.)

Reply:
fixed